src/flick/flick_auth/controllers/update_profile_controller.py

```diff
@@ -24,28 +24,34 @@ def process(self):
         email = self._data.get("email", "")
         bio = self._data.get("bio")
         phone_number = self._data.get("phone_number")
+        social_id = self._data.get("social_id")
         social_id_token = self._data.get("social_id_token")
         social_id_token_type = self._data.get("social_id_token_type")
 
-        if username and self._user.username != username:
+        if username is not None and self._user.username != username:
```

Comment:
ohh so doing just `if username`, it also false if it's an empty string?

Reply:
yahhhh